src/js/contentControl.js

```diff
@@ -17,10 +10,10 @@ module.exports = function() {
     function hideData() {
         // hide the data sections
         // these get shown as needed in addresses.js (render)
-        r.addClass('hide');
-        nR.addClass('hide');
-        nF.addClass('hide');
-        dup.addClass('hide');
+        $('#rural').addClass('hide');
+        $('#notRural').addClass('hide');
+        $('#duplicate').addClass('hide');
+        $('#notFound').addClass('hide');
```

Comment:
Ahhh I see. Typically saving the nodes is "better" because you aren't doing a lookup in the DOM every time you need to hide the data. But probably premature optimization at this point.